### ISSUE TITLE
jest occasionally tries to transform json files

### ISSUE THREAD

**Issue by smashercosmo** [jaredpalmer/tsdx#161]
### Current Behavior

Once in while I get the following warnings when I run 'tsdx test' command:
```
ts-jest[jest-transformer] (WARN) Got a unknown file type to compile (file: /Users/smashercosmo/Documents/projects/tmp-babel-plugin/node_modules/@babel/core/package.json). To fix this, in your Jest config change the `transform` key which value is `ts-jest` so that it does not match this kind of files anymore.
ts-jest[jest-transformer] (WARN) Got a unknown file type to compile (file: /Users/smashercosmo/Documents/projects/tmp-babel-plugin/node_modules/resolve/lib/core.json). To fix this, in your Jest config change the `transform` key which value is `ts-jest` so that it does not match this kind of files anymore.
ts-jest[jest-transformer] (WARN) Got a unknown file type to compile (file: /Users/smashercosmo/Documents/projects/tmp-babel-plugin/node_modules/globals/globals.json). To fix this, in your Jest config change the `transform` key which value is `ts-jest` so that it does not match this kind of files anymore.
```

### Expected behavior

jest shouldn't try transforming json files

### Suggested solution(s)

Probably 'json' should be added to transformIgnorePatterns in jest config

### Your environment

| Software         | Version(s) |
| ---------------- | ---------- |
| TSDX           | 0.7.2
| TypeScript       | 3.5.2
| npm         | 6.9.0
| Operating System | macOS Sierra 10.12.3

**Comment by jaredpalmer** [comment]
Is this still happening?

**Comment by smashercosmo** [comment]
I'll check

**Comment by smashercosmo** [comment]
Still getting this warnings with latest tsdx

**Comment by joemitchard** [comment]
I'm still finding this with v0.11.0. This usually occurs on the first run and then not again until the next build.

**Comment by SCasarotto** [comment]
I am still seeing this issues in v0.12.3

**Comment by agilgur5** [comment]
Fixed by #513 . It will be out in the next release
